modules/create.py

```diff
@@ -462,7 +463,8 @@ def addMovies(contentList, strm_name='', strm_type='Other', provider="n.a"):
                         if label and strm_name:                                              
                             label = str(utils.multiple_reSub(label.strip(), dictReplacements))
                             thisDialog.dialogeBG.update(j, ADDON_NAME + ": Gettin Movies: ",  " Video: " + label)
-                            movieList.append([os.path.join(strm_type), str(utils.multiple_reSub(label.strip(), dictReplacements)),link, listName])
+                            if filetype == 'file':
+                                movieList.append([os.path.join(strm_type, strm_name.strip().replace('++RenamedTitle++', '')), str(utils.multiple_reSub(label.strip(), dictReplacements)), link, listName])
```

Comment:
Ich glaube, wenn wir das machen, bekommen wir Probleme mit der Movies.db, es kann nur 1 STRM pro Film geben. Ansonsten stimmen die IDs in der DB nicht mehr. Außerdem werden dann Filme, in der Kodi-Library mehrfach angezeigt, ZB Avatar.strm von Amazon, xS, Exo usw, jede Avatar.strm wird dann in der Lybrary dargestellt.